README.md

```diff
@@ -1,42 +1,84 @@
   <!-- badges: start -->
-  [![CircleCI build status](https://circleci.com/gh/PolicyAnalysisInc/heRoMod.svg?style=svg)](https://circleci.com/gh/PolicyAnalysisInc/heRoMod)
- [![Coverage status](https://codecov.io/gh/PolicyAnalysisInc/heRoMod/branch/master/graph/badge.svg)](https://codecov.io/github/PolicyAnalysisInc/heRoMod?branch=master)
+
+[![CircleCI build status](https://circleci.com/gh/PolicyAnalysisInc/heRoMod.svg?style=svg)](https://circleci.com/gh/PolicyAnalysisInc/heRoMod)
+[![Coverage status](https://codecov.io/gh/PolicyAnalysisInc/heRoMod/branch/master/graph/badge.svg)](https://codecov.io/github/PolicyAnalysisInc/heRoMod?branch=master)
+
   <!-- badges: end -->
 
 # heRomod - A Package for Health Economic Modeling
 
-The heRomod package allows users to develop and run cost-effectiveness models in R and is the calculation engine used in the heRo web-based modeling platform.  A fork of the HEEMOD package.
+The heRomod package allows users to develop and run cost-effectiveness models in R and is the calculation engine used in the heRo web-based modeling platform. A fork of the HEEMOD package.
 
 You can install the latest released version from github with:
 
-```r
+## Developing Locally
+
+If you want to run the package locally from source
+
+1. Install R (>= 3.4.0) for your platform.
+    1. For Windows users, it is recommended to check Save version number in registry during installation so that the R extension can find the R executable automatically.
+    1. For mac users, use brew to install R and dependencies in terminal
+        1. `brew install R`
+        1. `brew install libgit2 harfbuzz fribidi freetype2 libpng libtiff libjpeg pandoc`
+1. Install languageserver in R.
+1. Open an R terminal
```

Comment:
It could be trivial, but I would leave the command to run: `r`.

In my case, I had problems because by default, `r` is a built-in command to re-run the last command. So after install `r` with `brew` I had to alias it:

```sh
alias r='/opt/homebrew/bin/r'
```

The path could vary depending on the installation, so I'd add a line to see it:

`whereis r`

---

Let me provide you a suggestion for this step:

```md
3. Open an R terminal executing `r` in your terminal
  - In some shells, `r` is a built-in command to re-run the last command.
     + You can verify it running `which r`.
     + If the response is `r: shell built-in command` you will need to alias it.
     + First check the path: `whereis r`
     + Then open you `rc` file (`.bashrc`, `.zshrc` or the one you use it) and add:
       `alias r='<your-path-to-r>'`.
     + For example: `alias r='/opt/homebrew/bin/r'`
     + Restart your shell
4. Run the following into your R terminal:
   ```r
   install.packages("languageserver")
   install.packages("usethis")
   install.packages("pkgdown")
   install.packages("devtools")
   ``
```